scripts/sbom_scraper.sh

```diff
@@ -160,16 +168,42 @@ then
 fi
 SECRET=$(cat "${CLIENTSECRET_FILE}")
 
+
 # ----------------------------------------------------------------------------
-# Extract SBOM
+# Deal with jar files - the argument should be of form
+# https://repo1.maven.org/maven2/org/assertj/assertj-core/1.0.0/assertj-core-1.0.0.jar
 # ----------------------------------------------------------------------------
-if [ "${SBOM}" = "false" ]
+if [ "${JARFILE}" = "true" ]
 then
-    log "Scrape ${FORMAT} SBOM from ${DOCKER_IMAGE} to ${OUTFILE} ..."
+    WORKDIR="${TEMPDIR}/jarfile"
+    mkdir "${WORKDIR}"
+    JARTYPE=$(echo "${DOCKER_IMAGE}" | cut -d':' -f1)
+    if [ "${JARTYPE}" = "https" ]
+    then
+        SUPPLIER_NAME=$(echo "${DOCKER_IMAGE}" | cut -d'/' -f4)
+        SUPPLIER_URL=$(echo "${DOCKER_IMAGE}" | cut -d'/' -f1-4)
+        (cd "${WORKDIR}" && curl -sSO "${DOCKER_IMAGE}")
+    else
```

Comment:
lets error out here

Reply:
moved logic to earlier in the script to error out before reaching here.